packages/demo-app/src/examples/CheckboxExample.tsx

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2021 Palantir Technologies, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+
+import * as React from "react";
+
+import { Checkbox, Radio } from "@blueprintjs/core";
+
+import { ExampleCard } from "./ExampleCard";
+
+export class CheckboxRadioExample extends React.PureComponent {
+    public render() {
+        return (
+            <div className="example-row">
+                <ExampleCard width={200}>
```

Comment:
nit: this looks a little awkward center aligned; can we make it left-aligned?

![image](https://user-images.githubusercontent.com/723999/134341574-a5acb1c7-ebba-4b52-985f-3b227374cbdd.png)